index.html

```diff
@@ -13,6 +13,73 @@
   </head>
 
   <body>
-   
+    <div class="container">
```

Comment:
- [OPTIONAL] This div here is not needed and you styled it with `width: 100vh`. this made your design unresponsive. Our design should be responsive design as it is a top requirement for this project. Kindly consider fixing it.
- Minimum mobile screen size should be `375`px. Errors in screen sizes less than `375`px should be ignored. Otherwise, mobile version should be responsive until the desktop screen size.